pkgs/desktops/deepin/library/gio-qt/default.nix

```
@@ -20,15 +22,23 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-dlY1CTlXywgGZUonBBe3cDwx8h2xXrPY6Ft/D59nlug=";
   };
 
-  nativeBuildInputs = [ cmake pkg-config wrapQtAppsHook ];
+  patchPhase = ''
```

```suggestion
  postPatch = ''
```

phases shouldn't be overriden